collection-scripts/gather_images

```diff
@@ -7,6 +7,7 @@ BASE_COLLECTION_PATH="/must-gather"
 IMAGES_PATH=${BASE_COLLECTION_PATH}/cluster-scoped-resources/images
 NAMESPACE_PATH=${BASE_COLLECTION_PATH}/namespaces
 
+mkdir ${BASE_COLLECTION_PATH}/cluster-scoped-resources
```

Comment:
pls use `mkdir -p`

Reply:
you can also pull ${BASE_COLLECTION_PATH}/cluster-scoped-resources into variable and use it for IMAGES_PATH prefix

Reply:
Done